.github/workflows/metric.yml

```diff
@@ -0,0 +1,71 @@
+name: Metric and Commit Results
+
+on:
+  workflow_dispatch: # Allows manual triggering of the workflow
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  compile-time:
+    name: Compile Time
+    runs-on: ubuntu-latest
+    # if: ${{ github.ref_name == 'main' }}
+    outputs:
+      time: ${{ steps.run.outputs.COMPILE_TIME }}
+    steps:
+      - uses: taiki-e/checkout-action@v1
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - uses: Boshen/setup-rust@main
```

Comment:
Need to setup warm cache here to avoid downloading crates.

Reply:
https://github.com/rolldown/rolldown/pull/1601/files#diff-34dda2f0b86ed8628bd9b41950351992acbe115340027caa42b64233895e17dcR24, did here.

Reply:
go check the logs if you aren't sure about the setup.

Reply:
https://github.com/rolldown/testcase/actions/runs/9919913382/job/27406237816, I test in `testcase` repo before.